Doc/library/string.rst

```diff
@@ -349,6 +364,11 @@ meaning in this case.
 The *sign* option is only valid for number types, and can be one of the
 following:
 
+   .. index::
+      single: +; in string formatting
+      single: -; in string formatting
+      single: space; in string formatting
```

Comment:
Sadly "space" goes to https://docs.python.org/dev/genindex-S.html as "space", not as a whitespace. But that's ok ;-)